build.sbt

```diff
@@ -11,6 +11,7 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 def buildLevelSettings: Seq[Setting[_]] =
   inThisBuild(
     Seq(
+      doc in Compile := file("/dev/null"),
```

Comment:
This should probably go.

Reply:
Yes. I keep accidentally pushing the commit that adds that.